pkg/vault/role/aws/aws.go

```diff
@@ -47,15 +47,15 @@ func NewAWSRole(kClient kubernetes.Interface, appClient appcat_cs.AppcatalogV1al
 
 }
 
-// If aws path does not exist, then use default database path
+// If aws path does not exist, then use default aws path
 func GetAWSPath(c appcat_cs.AppcatalogV1alpha1Interface, ref *appcat.AppReference) (string, error) {
 	vApp, err := c.AppBindings(ref.Namespace).Get(ref.Name, metav1.GetOptions{})
 	if err != nil {
 		return "", err
 	}
 
 	var cf struct {
-		AWSPath string `json:"aws_path,omitempty"`
```

Comment:
What defines this?

Reply:
It contains the path where aws secret engine is enabled